TopReadWidget/WMFTodayTopReadWidgetViewController.swift

```diff
@@ -284,9 +286,7 @@ class WMFTodayTopReadWidgetViewController: UIViewController, NCWidgetProviding {
         stackViewHeightConstraint.isActive = true
         
         stackViewHeightConstraint.constant = size.height
-        
-        view.layoutIfNeeded()
```

Comment:
Was the removal of this `layoutIfNeeded` call intentional?

Reply:
Initially yes but the refactor makes the removal no longer necessary